modules/local/multifasta_to_csv.nf

```diff
@@ -18,7 +18,8 @@ process MULTIFASTA_TO_CSV {
 
     script:
     """
-    echo -e id,sequence'\\n'${meta.id},`awk '!/^>/ {print \$0}' ${fasta} | tr '\\n' ':' | sed 's/:\$//'` > input.csv
+    awk '/^>/ {printf("\\n%s\\n",\$0);next; } { printf("%s",\$0);}  END {printf("\\n");}' ${fasta} > single_line.fasta
+    echo -e id,sequence'\\n'${meta.id},`awk '!/^>/ {print \$0}' single_line.fasta | tr '\\n' ':' | sed 's/:\$//' | sed 's/^://'` > input.csv
```

Comment:
No idea what this piece of code is doing and whether I would be able to update it myself 😅 But I trust you have tested yourself? Might be nicer to put complex things like this in a simple Python script or the likes.